pr_check.sh

```diff
@@ -1,18 +1,26 @@
 #!/bin/bash
 
-APP_NAME="rbac"  # name of app-sre "application" folder this component lives in
-COMPONENT_NAME="rbac"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
-IMAGE="quay.io/cloudservices/rbac"  # the image location on quay
+export APP_NAME="rbac"  # name of app-sre "application" folder this component lives in
+export COMPONENT_NAME="rbac"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+export IMAGE="quay.io/cloudservices/rbac"  # the image location on quay
 
-IQE_PLUGINS="rbac"  # name of the IQE plugin for this APP
-IQE_MARKER_EXPRESSION="rbac_smoke"  # This is the value passed to pytest -m
-IQE_FILTER_EXPRESSION=""  # This is the value passed to pytest -k
+export IQE_PLUGINS="rbac"  # name of the IQE plugin for this APP
+export IQE_MARKER_EXPRESSION="rbac_smoke"  # This is the value passed to pytest -m
+export IQE_FILTER_EXPRESSION=""  # This is the value passed to pytest -k
+export CJI_PATH="deploy/rbac-cji-smoketest.yml"
+
+find . -name cdappconfig.json
```

Comment:
What's this used for? :)

Reply:
Not sure. Seems like a debugging statement I inherited. Will remove